packages/dashboards/src/lib/configurator/components/heading/configurator-heading.component.ts

```diff
@@ -1,14 +1,21 @@
-import { Component, EventEmitter, Input, Output } from "@angular/core";
+import { Component, ElementRef, EventEmitter, Input, OnChanges, Output } from "@angular/core";
+import { ConfiguratorHeadingService } from "../../services/configurator-heading.service";
 
 @Component({
     selector: "nui-configurator-heading",
     templateUrl: "./configurator-heading.component.html",
     styleUrls: ["./configurator-heading.component.less"],
 })
-export class ConfiguratorHeadingComponent {
+export class ConfiguratorHeadingComponent implements OnChanges {
     @Input() public configuratorTitle: string;
     @Input() public disableCloseButton = false;
 
     // eslint-disable-next-line @angular-eslint/no-output-native
     @Output() public close = new EventEmitter();
+
+    constructor(public el: ElementRef, private heading: ConfiguratorHeadingService) {}
+
+    public ngOnChanges() {
+        this.heading.height$.next(+getComputedStyle(this.el.nativeElement).height.slice(0, -2));
```

Comment:
@dmytrofirsov, looking at this more closely, I'm not sure we should go this route. It seems non-ideal to have the configurator heading dictating the margins of the select components inside the dashwiz. Do you think we could enhance the edge detection service to somehow support edge detection for elements that are appended to body instead?

I think we should at least brainstorm together with @lukasholcik to see if there's a better way.

Reply:
Nevermind, Lukas tells me that a bunch of brainstorming has already been done.

Reply:
okay, thanks for the explanation